std-bits/table/src/main/java/org/enso/table/data/column/operation/unary/NumericUnaryOperation.java

```diff
@@ -0,0 +1,102 @@
+package org.enso.table.data.column.operation.unary;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import org.enso.table.data.column.builder.Builder;
+import org.enso.table.data.column.operation.StorageIterators;
+import org.enso.table.data.column.operation.UnaryOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemAggregator;
+import org.enso.table.data.column.storage.ColumnDoubleStorage;
+import org.enso.table.data.column.storage.ColumnLongStorage;
+import org.enso.table.data.column.storage.ColumnStorage;
+import org.enso.table.data.column.storage.NullStorage;
+import org.enso.table.data.column.storage.type.BigDecimalType;
+import org.enso.table.data.column.storage.type.BigIntegerType;
+import org.enso.table.data.column.storage.type.FloatType;
+import org.enso.table.data.column.storage.type.IntegerType;
+import org.enso.table.data.column.storage.type.NullType;
+
+public abstract class NumericUnaryOperation implements UnaryOperation {
+
+  @Override
+  public boolean canApply(ColumnStorage<?> storage) {
```

Comment:
In the [name of clarity](http://wiki.apidesign.org/wiki/ClarityOfAccessModifiers) shouldn't this method be `final`?
```suggestion
  public final boolean canApply(ColumnStorage<?> storage) {
```
This class is supposed to be subclassed and subclasses are supposed to override some methods. Certainly they have to implement `protected abstract` ones. Shall they also overide other methods like `canApply`? If not, prevent overriding.

Reply:
Hmm I imagine there may be situations like that, but in that case they'd also need to override `apply` - at which point there's little point in subclassing `NumericUnaryOperation` anymore (unless the override calls `super` in some cases).

I will make them `final` as suggested, we can always 'undo' it if need arises.